### PR TITLE
control-tags test update

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md
- [ ] This change has been tested via the terms in TESTING.md

### CODE REVIEW

File: cypress/integration/control-tags.spec.ts [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+      meridianMap.update({
+        tags: { showControlTags: true }
+      });
+    });
+
+    cy.get('[data-meridian-tag-id="546C0E014517"]').should("exist");
+    cy.get("[data-meridian-tag-id]").should("have.length", 4);
+  });
+
   it("control tags should be hidden by default", () => {
```
Comment (tarakreft) [inline cypress/integration/control-tags.spec.ts:27]: this is the same assertion as line 4 -- should this be "control tags should be showing by default"?

**Review by tarakreft — APPROVED** [review]

File: cypress/integration/control-tags.spec.ts (hunk 1 above)
Comment (vercasson) [reply cypress/integration/control-tags.spec.ts:27]: Totally! Thanks.

Comment (vercasson) [inline cypress/integration/control-tags.spec.ts:27]:
```suggestion
  it("control tags should be visible", () => {
```

File: cypress/integration/control-tags.spec.ts [hunk 2]
```diff
@@ -2,6 +2,29 @@ import { MeridianMap } from "../../src/web-sdk";
...
+
+    cy.get("[data-meridian-tag-id]").should("have.length", 3);
+    cy.get('[data-meridian-tag-id="546C0E014517"]').should("not.exist");
+
+    cy.window().then((contentWindow: any) => {
+      const meridianMap: MeridianMap = contentWindow.meridianMap;
+      meridianMap.update({
+        tags: { showControlTags: true }
+      });
+    });
```
Comment (wavebeem) [inline cypress/integration/control-tags.spec.ts:21]: Please refactor this to use the `getMeridianMap` helper 😄

**Review by wavebeem — APPROVED** [review]

**Pull request merged**